plugins/vpp/aclplugin/vppdump/dump_vppcalls.go

```diff
@@ -66,6 +66,9 @@ func DumpIPACL(swIfIndices ifaceidx.SwIfIndex, log logging.Logger, vppChannel VP
 
 	// get all ACLs with IP ruleData
 	IPRuleACLs, _ := DumpIPAcls(log, vppChannel, stopwatch)
+	if len(IPRuleACLs) < 1 {
+		return ACLs, nil
```

Comment:
You could just simply return `nil` instead of `ACLs` here and then also move the `ACLs` var right before the last loop, since it's not used anywhere before it.